### PR TITLE
Fix typos in the CFD-DEM parameters

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- There were some typos in the documentation

# Description of the solution

- Fixes typos and add hyperlinks between the pages.

### CODE REVIEW

File: doc/source/parameters/unresolved_cfd-dem/unresolved_cfd-dem.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
+The CFD-DEM solver templates are available in both 2D (``cfd_dem_coupling_2d``) and 3D (``cfd_dem>_coupling_3d``). The following parameter subsections are defined similarly to the CFD parameter template: "Simulation Control", "FEM", "Physical Properties" which represent the properties of the fluid, :doc:`../cfd/mesh/`, :doc:`../cfd/initial_conditions`, and :doc:`../cfd/boundary_conditions_cfd`. The following parameter subsections are defined similarly to the DEM parameter template: :doc:`../dem/model_parameters` and :doc:`../dem/lagrangian_physical_properties` which represents the physical properticles of the solid phase (particles). Two new subsections are introduced for the Volume Averaged Navier Stokes (VANS) solver as well as for the coupled unresolved CFD-DEM simulations. These are :doc:`void_fraction` and :doc:`cfd_dem` subsections. Parameters that are only applicable to the CFD-DEM solver are mentioned in their definitions.
```
Comment (lpsaavedra) [inline doc/source/parameters/unresolved_cfd-dem/unresolved_cfd-dem.rst:4]:
```suggestion
The CFD-DEM solver templates are available in both 2D (``cfd_dem_coupling_2d``) and 3D (``cfd_dem>_coupling_3d``). The following parameter subsections are defined similarly to the CFD parameter template: "Simulation Control", "FEM", "Physical Properties" which represent the properties of the fluid, :doc:`../cfd/mesh/`, :doc:`../cfd/initial_conditions`, and :doc:`../cfd/boundary_conditions_cfd`. The following parameter subsections are defined similarly to the DEM parameter template: :doc:`../dem/model_parameters` and :doc:`../dem/lagrangian_physical_properties` which represents the physical properties of the solid phase (particles). Two new subsections are introduced for the Volume Averaged Navier Stokes (VANS) solver as well as for the coupled unresolved CFD-DEM simulations. These are :doc:`void_fraction` and :doc:`cfd_dem` subsections. Parameters that are only applicable to the CFD-DEM solver are mentioned in their definitions.
```

File: doc/source/parameters/unresolved_cfd-dem/void_fraction.rst [hunk 2]
```diff
@@ -5,23 +5,23 @@ In this subsection, all parameters required for the calculation of the void frac
...
 
-     subsection function
-     set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
-     end
+   subsection function
+   set Function expression = 0.5 + 0.25*sin(pi*x)*sin(pi*y)
+   end
      
 If the ``mode`` chosen is ``dem``, then the void fraction is calculated using the Particle Centered Method. In this method, the remaining parameters are required.
 * The ``read dem`` allows us to read an already existing dem simulation result which can be obtained from checkpointing the Lethe-DEM simulation. This is important as the gls_vans solver requires reading an initial dem triangulation and particle information to simulate flows in the presence of particles.
```
Comment (lpsaavedra) [inline doc/source/parameters/unresolved_cfd-dem/void_fraction.rst:27]: There is something weird here. I think the bullet points are not working correctly? I'm not sure why?

**Review by lpsaavedra — COMMENTED** [review]
Good idea to change the location of the Compute Canada instructions. I found some typos and I have one comment. Also may I commit a small change adding a missing hyperlink in the "Launching your first simulation" section in this PR?

File: doc/source/parameters/unresolved_cfd-dem/void_fraction.rst (hunk 2 above)
Comment (blaisb) [reply doc/source/parameters/unresolved_cfd-dem/void_fraction.rst:27]: let me check it out

Comment (blaisb) [reply doc/source/parameters/unresolved_cfd-dem/void_fraction.rst:28]: Just fixed it in last commit ;).

Comment (lpsaavedra) [reply doc/source/parameters/unresolved_cfd-dem/void_fraction.rst:28]: I think a space between the sentence that ends with "required." and the one that starts with "* The ``read dem``" is missing.

**Comment by lpsaavedra** [comment]
Yes, I can approve. But I want to know if I can add in this PR some small commits where I fix some additional typos that I found in the documentation or if I should create a new PR?

**Comment by lpsaavedra** [comment]
I don't understand why the CI is failing...

**Comment by blaisb** [comment]
> I don't understand why the CI is failing...

The CI is failing most likely due to a particle change in the deal.II version, not because of you.
I'll merge anyway :).

**Comment by blaisb** [comment]
@lpsaavedra and @tonielgeitani  One of you must approve the PR, otherwise I can't merge it.
I can't approve my own PRs :)

**Review by lpsaavedra — APPROVED** [review]

**Pull request merged**